### ISSUE TITLE
Figure 2-Graphcoloring

### ISSUE THREAD

**Issue by chengquan50** [instadeepai/jumanji#274]
The reward of the graphcoloring environment is an integer, and the convergent mean generated by the three random seeds needs to be divisible by 3, but 3 times the convergent mean given in the graph is obviously not an integer.

**Comment by sash-a** [comment]
Hi @chengquan50 👋 

Not sure why the mean reward over 3 random seeds should be divisible by 3? The reward is simply `-num_colours_used` which may or may not be divisible by 3

**Comment by chengquan50** [comment]
My concern is that since reward is an integer in this environment, according to the legend, "Experiments were run with three different seeds, with the median represented as a blue curve and the min/max as the shaded region.", the blue curve should also be an integer, right? Thanks

**Comment by sash-a** [comment]
I see, it's possible that it was mean instead of median in this case by mistake, but I'm not sure. Maybe @clement-bonnet or @surana01 or @coyettev can give you a better answer as they worked on the graphs in the paper

**Comment by VitamintK** [comment]
The return of any one episode is an integer, but that doesn't mean that the expected return, even for a single random seed, is an integer. For example, in half of the problem instances, the agent may receive a return of 5, and in half it may receive a return of 6. The expected return is thus 5.5.

**Comment by clement-bonnet** [comment]
When used in training loops, we cast rewards as float in case they are not (or they are automatically casted). So the mean of the following reward sequence [5, 5, 6, 6] will therefore be 5.5.
